contrib/opencensus-ext-django/opencensus/ext/django/middleware.py

```diff
@@ -267,3 +272,29 @@ def process_response(self, request, response):
             log.error('Failed to trace request', exc_info=True)
         finally:
             return response
+
+    def process_exception(self, request, exception):
+        # Do not trace if the url is blacklisted
+        if utils.disable_tracing_url(request.path, self.blacklist_paths):
+            return
+
+        try:
+            if hasattr(exception, '__traceback__'):
+                tb = exception.__traceback__
+            else:
+                _, _, tb = sys.exc_info()
+
+            span = _get_django_span()
+            span.add_attribute(
+                attribute_key=ERROR_NAME,
+                attribute_value=repr(exception))
+            span.add_attribute(
+                attribute_key=ERROR_MESSAGE,
+                attribute_value=str(exception))
+            span.add_attribute(
+                attribute_key=STACKTRACE,
```

Comment:
This is a list. Should it be a list or a string with line breaks?

Reply:
I believe we should go with string with line breaks.